Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-@Library('pipeline-library') _
+@Library('pipeline-library@post-release-step-creds-for-github') _
```

Comment:
I'm testing this feature-branch out ([see diff here](https://bitbucket.org/inindca/pipeline-library/branches/compare/post-release-step-creds-for-github%0Dmaster#diff)). 

Currently, `shouldTagOnRelease` gets a permissions error from github when pushing tags. I think that pipeline-library should fix that issue. See the error at the end of [this build](https://jenkins.ininica.com/job/web-pipeline-deploy/594/consoleFull) for ewt.